crates/driver/src/tests/setup/blockchain/uniswap.rs

```diff
@@ -48,88 +48,123 @@ pub async fn setup() -> Uniswap {
         .balance(super::primary_address(&web3).await, None)
         .await
         .unwrap();
-    web3.eth()
-        .send_transaction(web3::types::TransactionRequest {
-            from: super::primary_address(&web3).await,
-            to: Some(admin),
-            value: Some(balance / 2),
-            ..Default::default()
-        })
-        .await
-        .unwrap();
+    super::wait_for(
```

Comment:
Wrapping every `send_tx` in a `super::wair_for` seems a bit tedious and indents every call one level to the right.
What do you think about using an extension trait for those calls to be able to do something like 👇 ?
```rust
web3.eth().send_transaction(Default::default()).wait_for_block(&web3).await;
```

Reply:
I just remembered that `ethcontract` supports transaction confirmations. You could write for example:
```rust
        authenticator
            .initialize_manager(admin)
            .from(admin_account.clone())
            .with_confirmations(0)
            .send(),
```